src/openstudio_lib/HVACSystemsController.cpp

```diff
@@ -358,37 +356,49 @@ void HVACSystemsController::update() {
         m_hvacSystemsView->mainViewSwitcher->setView(m_refrigerationGridController->refrigerationGridView());
       } else {
         // Not allowed
+        // TODO: should we automatically switch to the topologyView?
```

Comment:
I don't know if it would be more confusing to switch automatically or not, but this change to make it clear is at least an improvement.